### ISSUE TITLE
Draft participant handbook

### ISSUE THREAD

**Issue by dfm** [tess-science/online#3]
Some sections that need words:

- [x] **Goals**: Right now this is just text from the website. Let's be a bit more specific about what we want to achieve.
- [x] **What to expect**: broad brush strokes primer for (We will do pitch slides, you'll log in when you want in your time zone and you'll do XYZ, at the end of your session you'll do ABC before you log off)
- [x] **Schedule**: I think we've settled on something like:
  - [x] *coffee breaks* every 3 hours: some of these breaks will include activities designed with help from Charley
  - [x] *check-ins/report-backs/stand-ups* every ~6 hours: what should these look like?
  - [x] *research journals/check-out* whenever folks log off: this should include a form to provide anonymous feedback, but also a short summary of what everyone has been up to
  - [x] *final show-and-tell* at the end: What should this look like? One presentation per group? Videos? Slides? I'm probably inclined to aim for one slide per participant summarizing what they learned.
- [x] **Pitches**: We need to design a pitch template and describe it in the handbook. In particular, I think we want to encourage folks to consider how to make the projects *modular* and list specific sub-projects that can be assigned.
- [x] **How to collaborate remotely**: This should have some tips for tools and workflows. These will mostly just be recommendations, but I think we should require(?) that at least some of the key communication pass through Slack. Things like open google hangouts, remote pair coding, github projects, etc. @grd349: Can you take a look at this especially?
- [x] **How to lead a project**: I'd like this section to include some text about things that we expect from a project lead. In particular, encouraging them to make sure that they check in regularly with the other participants.

Other sections? Other thoughts?

**Comment by grd349** [comment]
Do we need a 'What to expect' section.  I.e., a broad brush strokes primer for (We will do pitch slides, you'll log in when you want in your time zone and you'll do XYZ, at the end of your session you'll do ABC before you log off).  It's possible this doesn't work but some sort of setting expectations so people are not intimidated by the format.  Make sense?

**Comment by dfm** [comment]
Yes - I love this! Adding it to the checklist above.

**Comment by grd349** [comment]
@dfm Yes - I will look at projects.  Current thoughts are something like 3 resources: using Git(hub) 101 clone/add/commit/push/pull; Contributing on GitHub issues/fork/PR/review/merge; Using GitHub projects to work with others.

The first 2 could be taken from external sources (plenty out there) and the third could be a medium length video.  Equally, the first two could be a video if all are keen.  (gotta practice online delivery as much as possible)

Does it make sense to have a slack channel for each 'tutorial' video to collect all questions?

**Comment by dfm** [comment]
I'm going to open a separate issue about collecting online resources because that's a great idea!

**Comment by dfm** [comment]
@benmontet: I made some (but not enough!) progress on this today. Any words, edits, suggestions you can add would be much appreciated!

**Comment by benmontet** [comment]
I think other sections we should have are "Code of Conduct" (which I wrote) and "Tips for First-Timers" (which I did not write). 

I also made some (but not enough) progress with words and will hand off. I think we can be finished in 24 hours if we all put a little time aside today/tomorrow!

**Comment by dfm** [comment]
We're getting there! The main sections that need to be filled in are:

1. The section describing pitches: what they are and how the discussion should proceed. There's some text in the Pitches document, but it needs to be expanded. I think that this is the key one we need to finish before sending invites.
2. The section with all the tips: I really don't feel qualified to do this, but I'm happy to start chipping at it later this evening. I think that the main one that needs to have something in it before we send out the invitations is the "Tips for First-Timers".

Otherwise I think we're in good shape!

**Comment by benmontet** [comment]
These words all look great! I've been in the unfortunate position of dealing with plagiarism cases all day (grades are due tomorrow) but that's now done so I get to do some writing

**Comment by dfm** [comment]
I'm going to close this now because we have a full draft. Let's open new issues if things need to be fixed!
